### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by Pakin-aw**
This feature is for adding an autocomplete into the search box.

### CODE REVIEW

File: lab-exercise-1/PakinFeature.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+  public class PakinFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (ACitronella) [inline lab-exercise-1/PakinFeature.java:3]: nice feature

Comment (TreephobKho) [inline lab-exercise-1/PakinFeature.java:3]: Please include this part.

File: lab-exercise-1/PakinFeature.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+  public class PakinFeature {
```
Comment (BunradarCH) [inline lab-exercise-1/PakinFeature.java:1]: good feature

File: lab-exercise-1/PakinFeature.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+      public void feature() {
+          // this is a new feature
+      }
```
Comment (sorrymaker1130) [inline lab-exercise-1/PakinFeature.java:4]: Good night

**Comment by Pakin-aw** [comment]
The new fix is available now!

File: lab-exercise-1/PakinFeature.java [hunk 4]
```diff
@@ -1,5 +1,7 @@
-  public class PakinFeature {
-      public void feature() {
-          // this is a new feature
-      }
-  }
+public class PakinFeature {
+    public void feature() {
+        // this is a new feature
+	int x = 0;
```
Comment (TreephobKho) [inline lab-exercise-1/PakinFeature.java:4]: Thank you for the edit.

File: lab-exercise-1/PakinFeature.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+	x = x + 1;
```
Comment (ACitronella) [inline lab-exercise-1/PakinFeature.java:5]: Cool change

File: lab-exercise-1/PakinFeature.java [hunk 6]
```diff
@@ -0,0 +1,7 @@
+public class PakinFeature {
+    public void feature() {
+        // this is a new feature
```
Comment (RolinAnn) [inline lab-exercise-1/PakinFeature.java:3]: Nice feature.

File: lab-exercise-1/PakinFeature.java (hunk 5 above)
Comment (sorrymaker1130) [inline lab-exercise-1/PakinFeature.java:5]: Add the part successful.

Comment (RolinAnn) [inline lab-exercise-1/PakinFeature.java:5]: Thank you for adding this part.

File: lab-exercise-1/PakinFeature.java [hunk 7]
```diff
@@ -0,0 +1,7 @@
+public class PakinFeature {
```
Comment (BunradarCH) [inline lab-exercise-1/PakinFeature.java:1]: good

**Pull request merged**